NuGet.Config

```diff
@@ -19,6 +19,7 @@
     <add key="nuget.org" value="https://www.nuget.org/api/v2/" />
     <add key="buildtools" value="https://www.myget.org/F/dotnet-buildtools/" />
     <add key="MyGetApplicationInsights" value="https://www.myget.org/F/applicationinsights" />
+    <add key="dotnet" value="https://dotnet.myget.org/F/dotnet-core/api/v3/index.json" protocolVersion="3" />
```

Comment:
Ok. Can you please create an issue similar to this https://github.com/Microsoft/ApplicationInsights-dotnet/issues/487 so we will not forget to fix it before 2.4 release